tests/test_rows.py

```diff
@@ -97,3 +97,37 @@ class Meta:
     tab = Tab([c])
     row = tab.rows[0]
     assert row.get_cell('a') == 'valA'
+
+
+def test_even_odd_css_class():
+    '''
+    Test for BoundRow.get_even_odd_css_class() method
+    '''
+    class SimpleTable(tables.Table):
+        foo = tables.Column()
+
+        def get_top_pinned_data(self):
+            return [{'foo': 'top-pinned'}]
+
+        def get_bottom_pinned_data(self):
+            return [{'foo': 'bottom-pinned'}]
+
+    data = [
+        {'foo', 'bar'},
+        {'foo', 'bas'},
+        {'foo', 'baz'},
+    ]
+
+    simple_table = SimpleTable(data)
+    count = 0
+
+    for row in simple_table.rows:
+        css_class = 'even' if count % 2 == 0 else 'odd'
```

Comment:
I'd say repeating the logic in a test is not the way to go. also: this test tests some different things as well. Maybe just check that the next value differs from the previous?